seqr/views/apis/users_api.py

```diff
@@ -78,9 +82,8 @@ def set_password(request, username):
         return create_json_response({}, status=400, reason='Password is required')
 
     user.set_password(request_json['password'])
-    user.first_name = request_json.get('firstName') or ''
-    user.last_name = request_json.get('lastName') or ''
-    user.save()
+    update_model_from_json(user, _get_user_json(request_json), user=user, updated_fields={'password'})
+    logger.info('Set password for user {}'.format(user.email), extra={'user': user})
```

Comment:
It technically doesn't but it seemed like a good idea to have extra logs related to login and logout and user management in general since those are the ones that are regularly called out in the FISMA compliance docs